giraffe/README.md

```diff
@@ -242,35 +247,31 @@ When using the comma separated values (CSV) from the Flux query as the `fluxResp
 ,,0,2020-03-25T20:58:15.731129Z,2020-04-24T20:58:15.731129Z,2020-04-10T22:20:40.776Z,28.7,value,temperature,index.html,browser
 `
 
-  const lineLayer = {
-    type: "line",
+  const lineLayer: LayerConfig = {
+    type: LayerTypes.Line,
     x: "_time",
     y: "_value",
+    fill: []
   }
 
-  const config = {
+  const config: Config = {
```

Comment:
We should avoid TypeScript in the examples. I find TypeScript to be an "advanced" tool. Our examples should be approachable. No need to confuse someone who just wants (and knows only how) to use React.